docs/excel/performance.md

```diff
@@ -62,22 +74,27 @@ await Excel.run(async (context) => {
 });
 ```
 
-Please note that only formula calculations are suspended. Any altered references are still rebuilt. For example, renaming a worksheet still updates any references in formulas to that worksheet.
+Only formula calculations are suspended. Structural updates, such as renaming a worksheet, still occur.
 
 ### Suspend screen updating
 
-Excel displays changes your add-in makes approximately as they happen in the code. For large, iterative data sets, you may not need to see this progress on the screen in real-time. `Application.suspendScreenUpdatingUntilNextSync()` pauses visual updates to Excel until the add-in calls `context.sync()`, or until `Excel.run` ends (implicitly calling `context.sync`). Be aware, Excel will not show any signs of activity until the next sync. Your add-in should either give users guidance to prepare them for this delay or provide a status bar to demonstrate activity.
+Excel displays changes as they occur. For large, iterative updates, suppress intermediate screen updates. `Application.suspendScreenUpdatingUntilNextSync()` pauses visual updates until the next `context.sync()` or the end of `Excel.run`. Provide your users with feedback such as status text or a progress bar, because the UI appears idle during suspension.
 
 > [!NOTE]
 > Don't call `suspendScreenUpdatingUntilNextSync` repeatedly (such as in a loop). Repeated calls will cause the Excel window to flicker.
 
 ### Enable and disable events
 
-Performance of an add-in may be improved by disabling events. A code sample showing how to enable and disable events is in the [Work with Events](excel-add-ins-events.md#enable-and-disable-events) article.
+You can sometimes improve performance by disabling events. A code sample showing how to enable and disable events is in the [Work with Events](excel-add-ins-events.md#enable-and-disable-events) article.
 
 ## Importing data into tables
 
-When trying to import a huge amount of data directly into a [Table](/javascript/api/excel/excel.table) object directly (for example, by using `TableRowCollection.add()`), you might experience slow performance. If you are trying to add a new table, you should fill in the data first by setting `range.values`, and then call `worksheet.tables.add()` to create a table over the range. If you are trying to write data into an existing table, write the data into a range object via `table.getDataBodyRange()`, and the table will expand automatically.
+When you import large datasets directly into a [Table](/javascript/api/excel/excel.table), such as repeatedly calling `TableRowCollection.add()`, performance can degrade. Instead, take the following approach:
+
+1. Write the entire 2D array to a range with `range.values`.
+2. Create the table over that populated range (`worksheet.tables.add()`).
+
+For existing tables, set values on `table.getDataBodyRange()` in bulk. The table expands automatically.
```

Comment:
I think this entire tip is a relic from ExcelApi 1.1. You can add multiple rows at once as of 1.4, which doesn't seem less efficient than the tips here.

Reply:
Do you think the entire "Importing data into tables" section can be removed from this Performance article?